allauth/account/forms.py

```diff
@@ -127,8 +134,9 @@ def __init__(self, *args, **kwargs):
             assert app_settings.AUTHENTICATION_METHOD \
                 == AuthenticationMethod.USERNAME_EMAIL
             login_widget = forms.TextInput(attrs={'placeholder':
-                                                  _('Username or e-mail'),
-                                                  'autofocus': 'autofocus'})
+                                                  _('Username e-mail'),
```

Comment:
Why did you remove "or" here?

Reply:
Probably unintentional. Feel free to restore the original string.